### DISCUSSION TITLE
Allow Youtube Videos with Adguard Home

### DISCUSSION THREAD

**Discussion by klmbrgr** [AdguardTeam/AdGuardHome#6454] [Q&A]
Hey guys,
I have Adguard Home in place for our family. Now Youtube recognizes Adguard Home and isn't displaying Videos anymore. For my use-case, I want to have Youtube enabled, even if it shows ads.
How do I configure that?
Thanks.

**Comment by Dynamic5912** [comment]
> Hey guys,
> 
> I have Adguard Home in place for our family. Now Youtube recognizes Adguard Home and isn't displaying Videos anymore. For my use-case, I want to have Youtube enabled, even if it shows ads.
> 
> How do I configure that?
> 
> Thanks.

Works fine here. Could it be because of a blocklist you're using?

**Comment by amrmzr** [comment]
try adding this custom filter https://raw.githubusercontent.com/hl2guide/AdGuard-Home-Whitelist/main/whitelist.txt as Allowlist. it should work fine.
